moveit_ros/moveit_servo/config/joystick_to_servo.yaml

```diff
@@ -0,0 +1,51 @@
+##########################################################################
+# Modify all parameters related to joystickk to servo here
+##########################################################################
+
+# Setup for Microsoft Xbox 360 Wired Controller
+enable_joint_commands_button: 4  # LB
+enable_twist_commands_button: 5  # RB
+switch_cartesian_command_frame_button: 1   # B
+switch_joint_axis_mode_button: 2  # X
+
+cartesian_command_frames:
+  - base_link
+  - tool0
+  - world
+
+joints:
+  - joint1
+  - joint2
+  - joint3
+  - joint4
+  - joint5
+  - joint6
+  - joint7
+
+# Do not change this names. The joints are order as in the list above
+num_joint_axis_control_modes: 2
```

Comment:
I think you could use a comment here. This confused me initially:

`# Each joystick axis has up to 2 different behaviors, depending on the control mode`